lib/api/controllers/authController.js

```diff
@@ -649,6 +678,151 @@ class AuthController extends NativeController {
       throw kerror.get('security', 'rights', 'unauthorized');
     }
   }
+
+  /**
+   * @param {Request} request
+   * @returns {Promise}
+   * @private
+   */
+  async _persistUser(request, profileIds, content, { humanReadableId=true } = {}) {
```

Comment:
You should avoid code duplication here since this method is also present in the `SecurityController`.  

Also, the `security.createRestrictedUser` method also contains duplicated code.